modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/token/TokenAPITestCase.java

```diff
@@ -69,8 +69,7 @@ public TokenAPITestCase(TestUserMode userMode) {
     @DataProvider
     public static Object[][] userModeDataProvider() {
         return new Object[][]{
-                new Object[]{TestUserMode.SUPER_TENANT_ADMIN},
-                new Object[]{TestUserMode.TENANT_ADMIN},
+                new Object[]{TestUserMode.SUPER_TENANT_ADMIN}
```

Comment:
Please make a note to add this back ASAP.

Reply:
Created an issue at https://github.com/wso2/product-apim/issues/3849